cms/src/main/java/uk/nhs/digital/externalstorage/resource/ImageDisplayPlugin.properties

```diff
@@ -0,0 +1,3 @@
+mimetype.label=Mime type:
```

Comment:
Does Mime Type mean anything to users? Would be better if we could show File Type (e.g extension) instead?

Reply:
This is 1:1 with the existing dialog.

but yes - we can make it more human friendly :)